### DISCUSSION TITLE
marketing opt-in

### DISCUSSION THREAD

**Discussion by ThomasGoatITMedia** [Codeinwp/otter-blocks#1618] [Ideas & Feedback]
Is it possible to have an "opt-in" checkbox where the user can check if they want to participate in the marketing options?

**Comment by HardeepAsrani** [comment]
@ThomasGoatITMedia Can you give us more details about it? Is it about a particular block, like Forms block?

**Comment by ThomasGoatITMedia** [comment]
Ofcourse!
It's in the forms block, there you can add form fields as well as set up a marketing part by adding a MailChimp api key.
My understanding is that it will allow you to create a form as well as let users subscribe to your newsletter.

It would be nice to have a checkbox where users actively opt-in for the MailChimp newsletter.
I'm using it as a contact form and I'd like users to opt-in for the newsletter

**Comment by Soare-Robert-Daniel** [comment]
@ThomasGoatITMedia, have you tried the option "Subscribe & Submit" in Marketing Integration? This option will render a checkbox for the user in which they can opt-in for saving their email in Mailchimp.

**Reply by ThomasGoatITMedia** [reply]
I did but it doesn't link to the privacy policy and I can't find where to change or translate the text

**Reply by Soare-Robert-Daniel** [reply]
The text is here https://github.com/Codeinwp/otter-blocks/blob/83941fab258dbf8211326dcd6eb34196461bb92b/inc/class-registration.php#L516

We will have checkboxes in the next update for Form, and I think we can iterate the idea where you can mark a checkbox field as privacy approval.
